src/chapters/html/background.rst

```diff
@@ -0,0 +1,100 @@
+When a user opens their browser to check out the programmer's web page and click on the URL, three main steps happen:
+
+1. The browser sends a **request** to the server for the web page.
+2. The server **responds** with the code that makes up the web page. 
+3. The browser takes the code and renders it to present the web page that the code creates. 
+
+When the browser renders the page, HTML outlines the structure of the page's content.
+
+.. admonition:: Note
+
+   In later chapters, request and response between browsers and servers will be covered in greater detail. 
+
+What is HTML?
+-------------
+
+.. index:: ! hypertext, ! markup language
+
+Indicators of how HTML works are in its name. HTML is short for Hypertext Markup Language.
+
+**Hypertext** is text that includes references to other text known as hyperlinks.
+
+With coding languages, there is a family of languages called **markup languages**. Markup languages annotate the text of a document and define the structure.
+HTML is the markup language that is defining the structure of hypertext.
+
+HTML's two main components are key to defining the structure of content: elements and tags.
+
+HTML Elements
+-------------
+
+.. index:: 
+   element
+
+When a programmer creates a web page, they break the content down by type.
+They may outline a structure for the page on paper first, highlighting what each item is.
+With HTML, a programmer can add a lot of different types of content to a page.
+In this chapter, the focus is on headings, paragraphs, images, and more.
+
+An **element** is a segment of an HTML page. Elements are oftentimes broken down by content type.
+
+HTML tags
+---------
+
+A HTML **tag** is the code that the computer processes to determine the type and content of an HTML element.
+
+Tags surround the content within the element, so in all cases, programmers need to have opening and closing tags.
+
+Each tag has the following structural elements:
+
+1. ``<`` to start a tag and ``>`` to close it.
+2. The type of element it is.
+3. Any extra information needed about the element.
+4. Closing tags include the same information as the opening tag with a ``/`` after the ``<`` bracket.
+
+Here is an example of a line of HTML:
+
+.. sourcecode:: html
+
+   <element type>content</element type> 
+
+HTML Writing Style
+------------------
+
+Programmers write HTML different ways with different style guides and philosophies.
+**Semantic HTML** helps programmers communicate through code and may be easier to pick up at first.
+Semantic HTML is not about the appearance of the web page, but about the specific meaning of the elements.
+Programmers can make a paragraph larger than a heading.
+But by looking at the HTML, another programmer can understand which is the paragraph and which is the heading.
+Another benefit to semantic HTML is that it is easier for beginning programmers to visualize the end results.
+Some examples of semantic HTML tags are: ``<p>``, ``<h1>``, ``<h2>`` , and ``<div>``.
```

Comment:
How do you feel about including additional examples like ``<footer>``, ``<section>``, ``<aside>``, etc?

Reply:
I didn't add those, because these are the tags that are covered later in the chapter.